.cargo/config.toml

```diff
@@ -1,2 +1,2 @@
-[build]
```

Comment:
This is effectively a noop change but it makes using a custom linker easier.  TL;DR rustflags are only cumulative within the same config section (i.e. target vs build).

And FWIW using mold has been very fast. Still wish the whole project would build faster but I'll take any improvements I can get.